code/log/lib/Loot/Log/Rio.hs

```diff
@@ -32,14 +32,14 @@ defaultLog l n t = do
 -- 'makeCap').
 defaultLogName
     :: forall m ctx.
-       (HasLens LoggingIO ctx LoggingIO, MonadReader ctx m, MonadIO m)
+       (HasLens' ctx LoggingIO, MonadReader ctx m, MonadIO m)
     => m NameSelector
 defaultLogName = view (lensOf @LoggingIO) >>= liftIO . _logName
 
--- | Default implementation of 'modifyLogNameSel' stack.
+-- | Default implementation of 'modifyLogNameSel' method.
```

Comment:
🤦‍♂️

Reply:
Oh ugh, I was removing something and then bringing it back :man_shrugging:

Reply:
It results in a proper fix.
Looks like if I hid ~a tresure~ piece of bullshit here and there and you constantly correct it 😺